pkg/engine/executor/executor.go

```diff
@@ -128,7 +129,16 @@ func (c *Context) executeDataObjScan(ctx context.Context, node *physical.DataObj
 		logsSection    *logs.Section
 	)
 
+	tenant, err := user.ExtractOrgID(ctx)
```

Comment:
I take it that the section index we store is still global to the object and not an index per tenant?

Reply:
Yes, the indexes store the number returned from Sections().Filter(), which is the object-global index.